debug.go

```diff
@@ -14,7 +14,7 @@ func (l *State) prototype(ci *callInfo) *prototype {
 	return l.stack[ci.function].(*luaClosure).prototype
 }
 func (l *State) currentLine(ci *callInfo) int {
-	return int(l.prototype(ci).lineInfo[ci.savedPC])
+	return int(l.prototype(ci).lineInfo[(ci.savedPC - 1)])
```

Comment:
From C Lua:
```c
#define pcRel(pc, p)	(cast(int, (pc) - (p)->code) - 1)
```
So yeah, the `- 1` looks correct. Thanks.

You don't need the extra parens.